### PR TITLE
Fixed #5562 -- Show welcome screen regardless of DEBUG value

### LINKED ISSUE #5562: Unable to create an initial page when DEBUG = False
With `DEBUG = False` whatever you do if you don't have any content in your database will lead you to a 404 page (even if you go to http://example.com/?edit), making you unable to do anything on your website. I'm not sure how I've never dealt with this issue before, having been a DjangoCMS user for quite some time now, so I might just be missing something. Anyway here's the code that fires up the "new page wizard" if there's no content and you're in DEBUG mode: https://github.com/divio/django-cms/blob/release/3.3.x/cms/page_rendering.py#L59. According to this code there's no way to get this wizard with debug mode disabled and without having any page in your database.

According to the docs (http://docs.django-cms.org/en/release-3.3.x/user/tutorial/index.html):

> When you visit a brand new site for the first time, you will be invited to log in.

I'd suggest to either fix these instructions to explain how to create the first page, or fix the code to make the process more user-friendly (I still haven't found a proper way to create my first page except enabling DEBUG on prod and then create the initial page).

### PR DESCRIPTION
**Pull request by czpython**
Fixed #5562 

* Changed signature of internal `_handle_no_page` function

### CODE REVIEW

**Comment by czpython** [comment]
@vxsx would you mind having a look? Seems like one of the FE tests is legitimately failing.

File: cms/page_rendering.py [hunk 1]
```diff
@@ -67,3 +60,10 @@ def _handle_no_page(request, slug):
         raise exc
     raise Http404('CMS Page not found: %s' % request.path)
 
+
+def _render_welcome_page(request):
+    context = {
+        'cms_version': __version__,
+        'cms_edit_on': get_cms_setting('CMS_TOOLBAR_URL__EDIT_ON'),
+    }
+    return TemplateResponse(request, "cms/welcome.html", context)
```
Comment (FinalAngel) [inline cms/page_rendering.py]: shouldn't that be single quotes?

**Comment by czpython** [comment]
This will land on 3.4.3

**Comment by czpython** [comment]
Because this is a change in behavior, I'm pushing this to 3.5.0

**Comment by edevil** [comment]
Since this only applies to 3.5.0, which has not been released yet, what is the correct way to bootstramp a CMS project?

Do we have to start it in DEBUG mode to add the first page?

**Comment by czpython** [comment]
You can add the first page directly in the admin.

**Pull request merged**